optionals/compat_xcam_taunus/config.cpp

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"xcam_taunus"};
```

Comment:
Is there an actual requirement? Seems like the config entry does not create one.

Maybe we should just have that in ACE3 directly if a compat is actually not needed.

Reply:
We should be able to just include the data, if the map exists or not
i.e. We're not adding it to cfgWorlds, but it's own custom config

Reply:
I'm not sure how much impact such big config entries could have on the performance/memory usage. Perhaps the data could be stored in the script file instead of config? With SQFC it should load fast enough.

Reply:
I will make a benchmark to see SQF vs SQFC vs config to see